src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,50 @@
+package model;
+
+public class FruitTransaction extends Fruit {
```

Comment:
Can we move the fields from `Fruit` here and remove the `Fruit` class to simplify your solution?
```suggestion
public class FruitTransaction {
```

Reply:
No, because I separate the transaction entity and fruit information to build the result file